demisto_sdk/__main__.py

```diff
@@ -1245,6 +1245,13 @@ def coverage_analyze(ctx, **kwargs):
     help="The path of the id_set json file.",
     type=click.Path(exists=True, resolve_path=True),
 )
+@click.option(
+    "-gr/-ngr",
```

Comment:
do we use `gr` as short for graph across multiple commands?

Reply:
We use it in validate